src/ng/animate.js

```diff
@@ -416,8 +416,8 @@ var $AnimateProvider = ['$provide', /** @this */ function($provide) {
        * @param {Promise} animationPromise The animation promise that is returned when an animation is started.
        */
       cancel: function(runner) {
-        if (runner.end) {
-          runner.end();
+        if (runner.cancel) {
+          runner.cancel();
```

Comment:
Because the error isn't coming from the splicer but from the cancel loop: http://plnkr.co/edit/MPUiYm4mTr9nNwg8SRS3?p=preview (Apologies if I'm being dense, it's very late again :D)